metadata-ingestion/src/datahub/ingestion/source/powerbi/rest_api_wrapper/powerbi_api.py

```diff
@@ -447,6 +473,16 @@ def fill_dashboards() -> None:
                 dashboard.tiles = self._get_resolver().get_tiles(
                     workspace, dashboard=dashboard
                 )
+                # set the dataset for tiles
+                for tile in dashboard.tiles:
+                    if tile.dataset_id:
+                        tile.dataset = self.dataset_registry.get(tile.dataset_id)
```

Comment:
is this dataset registry global across all workspaces?

also, do both workspaces need to be ingested for cross-workspace lineage to work? I think we might need some docs updates if so

Reply:
Yes both need to be ingested. You can check the info message at powerbi_api.py:176

Reply:
Ah I see - we should make this message way more clear then

it should say that this impacts lineage for the report/tile. the title should be like "missing lineage for report", and the context should say that it's like a cross-workspace reference that failed to be resolved